docs/vault/guide.md

```diff
@@ -119,6 +120,32 @@ You can monitor the operation of your Vault on the Vault dashboard by adding the
 
 Once the Vault is up and running, a "Vault" tab will appear in the topbar of the app at [testnet.interlay.io](https://testnet.interlay.io/) (or you can access directly at [testnet.interlay.io/vault](https://testnet.interlay.io/vault)).
 
+## Bitcoin Fees
+
+It is the responsibility of the Vault operator to ensure that they can cover excess Bitcoin fees when spending from additional UTXOs. To do this they can list addresses held by their wallet and transfer BTC from an [external source](https://bitcoin.org/en/exchanges).
```

Comment:
If we say it's their responsibility, then we should explain how they can check if they have enough. If you want to leave this for a future pr I'd be ok with that though

Reply:
I guess we need to link the Vault dashboard? I'll save that for a future PR